### PR TITLE
docs: replace unused variable by _

### PR DESCRIPTION
**Pull request by scizor666**
replace unused variable name heroes by _ in getHeroes() method

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
unused variable has the name "heroes"

Issue Number: N/A

## What is the new behavior?

unused variable name replaced by "_" like in the rest of the file for unused variables 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by scizor666** [comment]
I signed it!

File: aio/content/examples/toh-pt6/src/app/hero.service.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export class HeroService {
   // #enddocregion getHeroes-1
       .pipe(
         // #enddocregion getHeroes-2
-        tap(heroes => this.log('fetched heroes')),
+        tap(_ => this.log('fetched heroes')),
```
Comment (gkalpak) [inline aio/content/examples/toh-pt6/src/app/hero.service.ts]: If we're removing the var, it should be changed to `() =>`. (`_` will be a little confusing to beginners imo.)

Comment (scizor666) [reply aio/content/examples/toh-pt6/src/app/hero.service.ts]: if it's changed to `()` here I believe the same change will be required in lines 74, 88, 113, 123 then. I just wanted to make it consistent.  should we replace it to `()` everywhere?

Comment (gkalpak) [reply aio/content/examples/toh-pt6/src/app/hero.service.ts]: Oh, I didn't realize there were more occurrences of this pattern. Let's keep the `_` for consistency then 👍

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
@scizor666, can you, please, change the commit message to `docs: ...` (without the `(docs-infra)` part)?
(`docs-infra` is for changes that affect our docs app, tooling, or infrastructure, not actual docs changes.)

**Comment by scizor666** [comment]
> @scizor666, can you, please, change the commit message to `docs: ...` (without the `(docs-infra)` part)?
> (`docs-infra` is for changes that affect our docs app, tooling, or infrastructure, not actual docs changes.)

done

**Review by gkalpak — APPROVED** [review]

**Comment by jenniferfell** [comment]
@gkalpak @scizor666 I just took a look, because we went through the tutorial recently to verify against both v6 and v7. Looks like this change does not affect any surrounding text, correct? Are we sure this doesn't break any instructions or later use of the example? (not to be skeptical, but it's my nature with all TOH changes to ask a few extra questions. Thx!

**Review by jenniferfell — COMMENTED** [review]
See previous note. Thx.

**Comment by gkalpak** [comment]
> Looks like this change does not affect any surrounding text, correct?

I believe so :smiley:

> Are we sure this doesn't break any instructions or later use of the example?

Yes 💪 

##
@scizor666, can you also rebase this on master to fix the CI failure?

**Comment by scizor666** [comment]
>> @scizor666, can you also rebase this on master to fix the CI failure?

@gkalpak done

**Comment by gkalpak** [comment]
Ha ha, we got new failures now 🤦‍♂️ 
Note to caretaker: CI failures are unrelated 🙇

**Pull request closed without merging**